diff -Naur a/configure b/configure
--- a/configure	1970-01-01 00:00:00.000000000 +0000
+++ b/configure	1970-01-01 00:00:00.000000000 +0000
@@ -4909,9 +4909,7 @@
   ac_prog_version=`$CC -v 2>&1 | sed -n 's/^.*version \([egcygnustpi-]*[0-9.]*\).*$/\1/p'`
   case $ac_prog_version in
     '') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
-    4.[3-9].* | 4.[1-9][0-9].* | [5-9].* )
-       ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
-    *) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
+    *) ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
 
   esac
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_prog_version" >&5
@@ -4972,9 +4970,7 @@
   ac_prog_version=`$MAKE --version 2>&1 | sed -n 's/^.*GNU Make[^0-9]*\([0-9][0-9.]*\).*$/\1/p'`
   case $ac_prog_version in
     '') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
-    3.79* | 3.[89]*)
-       ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
-    *) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
+    *) ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
 
   esac
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_prog_version" >&5
